pkgs/test_core/lib/src/runner/engine.dart

```diff
@@ -220,7 +213,6 @@ class Engine {
   /// four times [concurrency].
   Engine({int concurrency, int maxSuites, String coverage})
```

Comment:
We should remove the `maxSuites` argument since it has no meaning anymore. Was that used anywhere?

Reply:
It wasn't used within any of the `test` packages.

It is possible it was used elsewhere but it is somewhat difficult to tell for sure.

We are in a breaking release anyways so I will go ahead and just delete it.